package.json

```diff
@@ -183,7 +184,6 @@
     "percy": "^0.28.6",
     "prettier": "^2.2.1",
     "puppeteer": "^5.3.1",
-    "react-test-renderer": "^16.14.0",
```

Comment:
Why was this removed?

Reply:
This is used by Testing Library and should definitely stay in the main `package.json` file.